### DISCUSSION TITLE
Open source playground

### DISCUSSION THREAD

**Discussion by vhpoet** [openfga/community#47] [Q&A]
OpenFGA announcement [mentions](https://auth0.com/blog/auth0s-openfga-open-source-fine-grained-authorization-system/#:~:text=Auth0%E2%80%99s%20FGA%20Playground%2C%20which%20is%20not%20open%20source%20now%20but%20it%20will%20be%20in%20the%20following%20weeks.) that the playground is going to be open sourced. Any idea when is this going to happen?

**Comment by aaguiarz** [comment]
Hi @vhpoet,

We are targeting it by the end of the year, but that might change. 

We can currently use the https://play.fga.dev site to point to an OpenFGA instance to manage it, and that has been functionally good enough to work with OpenFGA.

Do you have a different use case in mind for it?

Thanks!

**Comment by vhpoet** [comment]
Hey Andrés, thanks for the quick response. I wanted to point it to a deployed instance, but I get a

```
Refused to connect to 'http://DEPLOYED_INSTANCE_DOMAIN/stores' because it violates the following Content Security Policy directive: "connect-src 'self' https://api.playground.fga.dev http://127.0.0.1:* http://localhost:* https://cloudflareinsights.com https://cdn.cookielaw.org https://privacyportal.onetrust.com https://heapanalytics.com".
```

**Comment by rhamzeh** [comment]
Thanks for raising this issue @vhpoet - May I ask if the DEPLOYED_INSTANCE_DOMAIN is on your internal network?

Would allowing it to talk to internal networks suffice? Otherwise we'll see if we can drop the CSP for the time being, or offer an automated method to have it work for you.

Note: **Make sure DEPLOYED_INSTANCE_DOMAIN is not publicly accessible**, the Playground is not meant to be publicly exposed at it can be used to manipulate your data.

**Comment by vhpoet** [comment]
Hey @rhamzeh, the deployed instance is not public (can only be access through a VPN). What do you mean by allowing it to talk to internal networks? How would that work?
